src/CSI.IBTA.BenefitsService/Controllers/InsurancePlansController.cs

```diff
@@ -51,7 +52,7 @@ public async Task<IActionResult> GetPlan(int planId)
         }
 
         [HttpGet("PlanTypes")]
-        [Authorize]
+        [Authorize(Roles = $"{nameof(Role.Administrator)}, {nameof(Role.EmployerAdmin)}, {nameof(Role.Employee)}")]
```

Comment:
Is actually necessary to specify every role instead of just keeping [Authorize]?